nise-populator/sources/oci.py

```diff
@@ -20,10 +20,12 @@ class OCI(Source):
 
     def __init__(self, **kwargs):
         """Initialize the source with configuration data."""
+        super().__init__(**kwargs)
         self.kwargs = kwargs
-        self.static_file = kwargs.get(self.STATIC_FILE)
+        self.oci_static_file = kwargs.get(self.STATIC_FILE)
```

Comment:
Nit: This is a static file associated with an OCI instance. Putting `oci_` in the variable name is redundant.

Reply:
Yes, it seems a bit redundant. I got thrown off by a `sonarlint`

Since its not a blocker, I'll defer cleaning this one up for later.